src/rag_processor/api/batch.py

```
@@ -0,0 +1,194 @@
+"""Batch and job status API endpoints.
+
+Provides endpoints for querying batch and job status.
+"""
+
+from __future__ import annotations
+
+from uuid import UUID  # noqa: TC003 - Used at runtime by FastAPI
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `TC003`)

Remove unused `noqa` directive

(RUF100)